portal/models/procedure_codes.py

```diff
@@ -158,6 +192,15 @@ def OtherProcedure(self):
             codings=[sno, ichom], text='Other procedure on prostate'
         ).add_if_not_found(True)
 
+    @lazyprop
+    def NoneOfTheAbove(self):
```

Comment:
Lets change the property name to `OtherPrimaryTreatment` (snomed, 999999999), if we need this one, so it doesn't collide with the pre-existing `NoneOfTheAbove` (truenth, 999).